packages/client/src/api/types.ts

```diff
@@ -0,0 +1,5 @@
+export type responseData = Record<string, never>;
+
+export type setUserThemeData = {
```

Comment:
тайпинги принято называть с заглавной буквой
```suggestion
export type ResponseData = Record<string, never>;

export type SetUserThemeData = {
```